sdk/src/main/scala/io/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -17,7 +18,7 @@ object AccountFeePaymentsUtils {
     }
   }
 
-  def getForgersRewards(blockFeeInfoSeq : Seq[AccountBlockFeeInfo]): Seq[AccountPayment] = {
+  def getForgersRewards(blockFeeInfoSeq : Seq[AccountBlockFeeInfo], mcForgerPoolRewards: Map[AddressProposition, BigInteger] = Map.empty): Seq[AccountPayment] = {
     if (blockFeeInfoSeq.isEmpty)
       return Seq()
```

Comment:
Changed this condition, so that if blockFeeInfoSeq is empty, we just take mcForgerPoolRewards and turn them into AccountPayment. If mcForgerPoolRewards is also empty, this would return empty Seq